### PR TITLE
Yaml

### PR DESCRIPTION
**Pull request by NathanTP**
I'm tired of not being able to comment stuff out. YAML is a bit nicer to work with anyway. I don't anticipate using any fancy features, and JSON will remain supported.

### CODE REVIEW

File: scripts/fullTest.py [hunk 1]
```diff
@@ -126,7 +126,7 @@ def runTests(testNames, categoryName, marshalArgs=[], cmdArgs=[]):
 
     for tName in testNames:
         log.log(logging.INFO, "[{}] {}:".format(categoryName, tName))
-        tPath = testDir / (tName + ".json")
+        tPath = testDir / (tName + ".yaml")
```
Comment (a0u) [inline scripts/fullTest.py:129]: It seems that all `test/*.json` files should be renamed to `*.yaml`, but they don't appear to be committed yet.  I see a bunch of file deletions in the diff but no moves.

Comment (NathanTP) [reply scripts/fullTest.py:129]: derp. fixed

File: wlutil/config.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import glob
 import collections
 import json
```
Comment (a0u) [inline wlutil/config.py:3]: Is this import still necessary?

**Review by a0u — DISMISSED** [review]
LGTM.

File: wlutil/config.py (hunk 2 above)
Comment (NathanTP) [reply wlutil/config.py]: nope. In fixing that I also found a bug in the exception handler for invalid yaml files. Anyway, both fixed now.

**Review by a0u — APPROVED** [review]

**Pull request merged**